frontend/packages/topology/src/components/quick-search/QuickSearchDetails.scss

```diff
@@ -0,0 +1,35 @@
+.odc-quick-search-details {
+  display: flex;
+  flex-direction: column;
+  padding: 6px 12px 0px 15px;
+  overflow-y: auto;
+  overflow-wrap: break-word;
+  width: 98%;
```

Comment:
Hmmm, 98% is fishy... what was this in regards to?

Reply:
I removed this and it appears to work fine... what was the intent? (is it browser conformance? I only tested on Chrome)

Reply:
So I have kept the inner containers width to be 98% because I wanted a space between the parent container and where the scrollbar appears for the inner container. If I remove the width the space wont be there anymore as shown in the UX doc. Any suggestions?
![image](https://user-images.githubusercontent.com/20724543/100368398-8c442280-3029-11eb-9d14-d05d8b0ea13b.png)

Reply:
Interesting... yeah it does appear like that in the design. Padding is a better way to go here... `width` of a % can be misleading and less extensible. Really what you're looking for is the gap, not a percentage of the width. If we made the modal wide or dynamically scaled it, % means something misleading for what we want... padding is preferred here.